charm-slurmd/config.yaml

```diff
@@ -27,8 +27,11 @@ options:
     type: string
     default: ""
     description: >
-      Extra partition configuration specified as a space separated `key=value`
-      single line.
+      Extra partition configuration, specified as a space separated `key=value`
+      in a single line.
+
+      Example usage:
+      $ juju config slurmd partition-config="DefaultTime=45:00 MaxTime=1:00:00"
```

Comment:
Do you think it is worth linking to https://slurm.schedmd.com/slurm.conf.html#SECTION_PARTITION-CONFIGURATION

Reply:
It may just clutter the description. Just a thought.

Reply:
We could link that in the docs: https://omnivector-solutions.github.io/osd-documentation/master/operations/partitions.html

Reply:
Done: https://github.com/omnivector-solutions/osd-documentation/pull/44